src/components/Preview.jsx

```diff
@@ -7,14 +7,18 @@ import classnames from 'classnames';
 import generatePreview from '../util/generatePreview';
 import {openWindowWithWorkaroundForChromeClosingBug} from '../util';
 import PreviewFrame from './PreviewFrame';
+import {
+  setRefreshTimestamp
+} from '../actions';
 
 class Preview extends React.Component {
   constructor() {
     super();
     bindAll(this, '_handlePopOutClick');
+    bindAll(this, '_handleRefreshClick');
   }
 
-  _generateDocument(isLivePreview = false) {
+  _generateDocument(isLivePreview = false, timestamp = this.props.lastRefreshTimestamp) {
```

Comment:
I don’t think there’s any need for `timestamp` to be a parameter here—seems like it will always come from the `props`?